### DISCUSSION TITLE
Build Failed

### DISCUSSION THREAD

**Discussion by AnitaAO** [vercel/community#1239] [Help] [answered]
### Summary

I am new to Next, I deployed my project and in the first two days, the builds were successful. I cannot explain what later went wrong because now all my builds fail with the error message ' "npm run build" exited with 1 '.

Please I need help
![Screenshot (68)](https://user-images.githubusercontent.com/37036097/211604195-c365af59-858a-40e2-8e44-84a7aa7646cd.png)

### Additional information

13:47:56.146 | Cloning completed: 629.39ms
13:48:00.131 | Restored build cache
13:48:00.169 | Running "vercel build"
13:48:00.670 | Vercel CLI 28.10.3
13:48:00.992 | Installing dependencies...
13:48:02.932 |  
13:48:02.933 | changed 1 package in 2s
13:48:02.933 |  
13:48:02.933 | 125 packages are looking for funding
13:48:02.933 | run `npm fund` for details
13:48:02.942 | Detected Next.js version: 13.0.5
13:48:02.946 | Detected `package-lock.json` generated by npm 7+...
13:48:02.947 | Running "npm run build"
13:48:03.248 |  
13:48:03.249 | > nvrates-web@0.1.0 build
13:48:03.249 | > set "CI=false" && next build
13:48:03.249 |  
13:48:03.563 | info  - Loaded env from /vercel/path0/.env
13:48:03.817 | info  - Linting and checking validity of types...
13:48:10.903 | info  - Creating an optimized production build...
13:48:11.191 | info  - Disabled SWC as replacement for Babel because of custom Babel configuration ".babelrc" https://nextjs.org/docs/messages/swc-disabled
13:48:11.889 | info  - Using external babel configuration from /vercel/path0/.babelrc
13:48:27.345 | info  - Compiled successfully
13:48:27.345 | info  - Collecting page data...
13:48:33.168 |  
13:48:33.168 | > Build optimization failed: found page without a React Component as default export in
13:48:33.169 | pages/types
13:48:33.169 |  
13:48:33.169 | See https://nextjs.org/docs/messages/page-without-valid-component for more info.
13:48:33.169 |  
13:48:33.227 | Error: Command "npm run build" exited with 1

**Comment by swarnava** [comment, marked as answer]
Hi there,

```
> Build optimization failed: found page without a React Component as default export in pages/types

See https://nextjs.org/docs/messages/page-without-valid-component for more info.
```

 You can move `types.ts` out of the `pages` dir if you dont intend for that to become a page that can be visited at `/types` route
